### ISSUE TITLE
Tilt support

### ISSUE THREAD

**Issue by dadosch** [xournalpp/xournalpp#2065]
**Is your feature request related to a problem? Please describe.**
Wacom tablets, together with supported pens, suport the recognition of the current tilt of the pen (in X and in Y direction). Currently, this tilt isn't considered.

**Describe the solution you'd like**
It would be nice, if the tilt would have some effect on how the stroke is: if the pen is tilted, it should affect the stroke with. That is, if the pen is in a 90° angle (in X and y direction), the stroke should the most think one. The more the pen is tilted, the more the stroke should be thinner. (I think that is how it should be?)

**Describe alternatives you've considered**

**Additional context**
You can see if your pen supports tilt, by running `evtest | grep TILT_X` and then selecting your wacom pen input device.

**Comment by LittleHuba** [comment]
Tilt is not supported by GTK+. It does not provide data for tilt. You have to request the feature there first.

**Comment by dadosch** [comment]
I think this issue: https://gitlab.gnome.org/GNOME/gtk/-/issues/1392
which was closed in favor of this one: https://gitlab.gnome.org/GNOME/gtk/-/issues/1551

tracks the support for tilt.

**Comment by lb90** [comment]
Hi @LittleHuba, just wanted to say that in GTK3 tilt is supported :) See https://developer.gnome.org/gdk3/stable/GdkDevice.html#GdkAxisUse

**Comment by LittleHuba** [comment]
Yes they do say it is supported. Reading the data consistently gives either "0" or "1". Don't remember which, but it does not actually give you any real values. The problem is that GDK+ does not read the values from the device but GTK provides the API to request the data.

**Comment by lb90** [comment]
You surely have much more experience than me, and I'm not that much into graphics. Anyway, GIMP developers say that tilt is [working](https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/1563#note_1179095) for them! Could it be just a specific issue with your devices?

Thank you!

**Comment by LittleHuba** [comment]
It could be that this was fixed recently. I checked this over a year ago. So it could definitely be that this was fixed since then.

**Comment by lylythechosenone** [comment]
Any update on this?
